bundles/fr.adaussy.permadeler.model.design.tests/src/fr/adaussy/permadeler/model/design/services/ValidationServiceTest.java

```diff
@@ -0,0 +1,211 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Arthur Daussy.
+ *
+ * This program and the accompanying materials are made 
+ * available under the terms of the Eclipse Public License 2.0 
+ * which is available at https://www.eclipse.org/legal/epl-2.0/ 
+ * Contributors:
+ * Arthur Daussy - initial API and implementation.
```

Comment:
Change "Arthur Daussy" to "Lucas Chaigne"

Reply:
done